app/viewmodels/models/YesNoPageViewModel.scala

```diff
@@ -24,6 +24,7 @@ case class YesNoPageViewModel(
   heading: SimpleMessage,
   description: List[SimpleMessage],
   legend: Option[SimpleMessage],
+  inline: Boolean = true,
```

Comment:
I spoke with Josh about how some yes no pages are inline and some are not, he said he was going to change the designs so none were inline, perhaps drop him a message and ask, we may be able to remove it